CONTRIBUTING.md

```diff
@@ -32,181 +32,14 @@ npx nx run-many -t integration-test
 npx nx affected:lint
 ```
 
-## Lint
+## Linting
 
-### Configuration
-
-This projects has a default linting target `lint`. Every task has 0 configured as `maxWarnings`.
-
-You can find more information on our [EsLint strategy](./tools/scripts/eslint-migration-next/docs/eslint-strategies.md) in the docs.
-
-```json
-{
-  "targetDefaults": {
-    "lint": {
-      "options": {
-        "maxWarnings": 0
-      }
-    }
-  }
-}
-```
-
-### Violation Categories
-
-This repository has a strict categorisation of lint issues:
-
-- **warnings** - code style only
-- **problems** - bug prevention only
-
-### Config Strategy
-
-This repo maintains a `Root-Level General Rules with Extensions in Packages` EsLint config strategy.
-
-```shell
-project-root/
-  ├── eslint.config.js
-  └── packages/
-      ├── frontend/
-      │   └── eslint.config.js
-      └── backend/
-          └── eslint.config.js
-```
-
-#### Configuration
-
-`eslint.config.js`
-
-```javascript
-module.exports = {
-  rules: {
-    'no-console': 'warn',
-    'no-unused-vars': 'error',
-  },
-};
-```
-
-`packages/frontend/eslint.config.js`
-
-```javascript
-const rootConfig = require('../../eslint.config');
-
-module.exports = {
-  ...rootConfig,
-  rules: {
-    ...rootConfig.rules,
-    'no-console': 'off', // Override root rule
-  },
-};
-```
-
-### EsLint Migration
-
-We run the "[eslint next](./tools/scripts/eslint-next/README.md)" migration strategy.
-That means we maintain a second config that maintains the target rules `eslint.next.config.js` and a default config `eslint.config.js` that has all faining rules enabled.
-
-After every change on EsLint rule migrations run the update script to keep your configurations automatically in sync.
-
-```sh
-pnpm `eslint-next/nx`
-```
-
-This will update all projects.
-
-```sh
-pnpm `eslint-next/nx --projects=build`
-```
-
-For subset of projects.
-
-For more detail read the [`eslint-next` docs](./tools/scripts/eslint-next/README.md).
+- The docs are located in the [./docs/linting/eslint.md](./docs/linting/eslint.md) folder.
```

Comment:
Will you hate me if I ask you to rename this folder to something akin to "repo-docs" or some other name to distinguish it from the docs site (which is `apps/docs`)

I don't want people confused as to where they can find the docs for repo and docs site for the plugins themselves.

Reply:
> Will you hate me if
Not for this request ;) 

Sure! what about `contributing` ? e.g. `contributing/linting/eslint.md`?

Reply:
yeah that works for me!

Reply:
done